content/docs/command-reference/fetch.md

```diff
@@ -228,53 +228,51 @@ the cache entries for the `data/features` directory, and
 ## Example: With dependencies
 
 After following the previous example (**Specific stages**), only the files
-associated with the `prepare.dvc` stage file have been fetched. Several
+associated with the `prepare` stage has been fetched. Several
 dependencies/outputs of other pipeline stages are still missing from the cache:
 
 ```dvc
 $ dvc status -c
 ...
-    deleted:            model.pkl
     deleted:            data/features/test.pkl
     deleted:            data/features/train.pkl
-    deleted:            data/data.xml
+    deleted:            model.pkl
 ```
 
 One could do a simple `dvc fetch` to get all the data, but what if you only want
-to retrieve the data up to our third stage, `train.dvc`? We can use the
+to retrieve the data up to our third stage (`train`)? We can use the
```

Comment:
OK to remove ".dvc" but why did we change comma for parentheses?

Reply:
Changed.

Reply:
Now it's missing the comma... Please let's not overcomplicate these simple replacing tasks unless there's a strong reason to change existing syntax. 3 rounds of feedback on a single replace shouldn't be needed. Thanks